codeface_extraction/extractions.py

```diff
@@ -14,13 +14,18 @@
 #
 # Copyright 2015-2018 by Claus Hunsen <hunsen@fim.uni-passau.de>
 # Copyright 2016 by Thomas Bock <bockthom@fim.uni-passau.de>
+# Copyright 2018 by Barbara Eckl <ecklbarb@fim.uni-passau.de>
+# Copyright 2018 by Tina Schuh <schuhtb@fim.uni-passau.de>
```

Comment:
I just recognized that the "b" in Tina's e-mail address is wrong. Should be removed, I guess.